packages/engine/vscode/src/lib/compile_worker.ts

```diff
@@ -0,0 +1,118 @@
+import * as solc from "solc";
+import * as path from "path";
+import * as fs from "fs";
+import axios from "axios";
```

Comment:
If possible don't add unnecessary external dependencies. You can reuse the `fetch` function in webview.ts of the same folder. It looks like that : 
```typescript
/** Fetch remote ressource with http */
function fetch(url: string): Promise<string> {
  return new Promise((resolve, reject) => {
    get(url, res => {
      let text = ''
      res.on('data', data => text += data)
      res.on('end', _ => resolve(text))
      res.on('error', err => reject(err))
    })
  })
}
```